… and it breaks our Windows MSVC CI builds, e.g.:
```
▾ Build the main module
  make -C ./configure install
  ../configure/RULES_ARCHS:67: *** Base not configured to build target windows-x64.  Stop.
  make[1]: Entering directory 'D:/a/epics-base/epics-base/configure'
  make[1]: Leaving directory 'D:/a/epics-base/epics-base/configure'
  make: *** [configure/RULES_DIRS:85: configure.install] Error 2
  Error: Process completed with exit code 1.
```

I was wrong about the file `$(CONFIG)/os/CONFIG.Common.$(T_A)`, it isn't required, only `$(CONFIG)/os/CONFIG.$(EPICS_HOST_ARCH).Common` is essential. Bummer.

Would there be objections to my creating those missing files and making them required? That would obviously require a Release Note warning sites they would need to add them for their local architectures too.

--- a/configure/RULES_ARCHS
+++ b/configure/RULES_ARCHS
@@ -64,6 +64,9 @@ $(BUILD_ARCHS) : % : O.% O.Common
 $(ACTIONS) : % : $(foreach arch, $(BUILD_ARCHS), %$(DIVIDER)$(arch))
 
 $(buildDirs):
+	$(if $(wildcard $(CONFIG)/os/CONFIG.Common.$(archPart) \
+	        $(CONFIG)/os/CONFIG.$(EPICS_HOST_ARCH).$(archPart)),, \
+	    $(error Base not configured to build target $(archPart)))
 	$(PERL) $(TOOLS)/makeMakefile.pl $@ $(TOP)/..
 
 O.Common:

